### DISCUSSION TITLE
Amazon and Druid classes

### DISCUSSION THREAD

**Discussion by Despex94** [isid0re/SoloLeveling#31] [General]
Hi, isid0re, your script can play Amazon and Druid classes?

**Comment by isid0re** [comment]
yes, I am currently working out the various start builds for both. I expect to have a basic outline up and running before years end for the druid and assassin.

**Comment by zanrub** [comment]
Hey isid0re, i started using your bot a week ago on a sorc and everything works really smooth. Thanks a lot for building the bot!
Is there any progress regards the amazon build?

**Reply by isid0re** [reply]
There is an amazon coming. I dont have an ETA. Right now I need to focus on a few mechanical problems with the bot. For example, its not currently keeping found polearms and instead only buying them. Its been inconsistent on wearing dual rings. So issues like these are what I want  to resolve before i start adding more builds.

**Reply by zanrub** [reply]
Thanks for your reply. I also had the problem with the 2nd ring. Thanks again for all the work you put into this project.

**Reply by theBGuy** [reply]
I am working on the amazon it will be released soon
